lib/open_project/backlogs/patches/specific_work_package_schema_patch.rb

```diff
@@ -33,12 +33,12 @@
 # See doc/COPYRIGHT.rdoc for more details.
 #++
 
-require_dependency 'api/v3/work_packages/schema/work_package_schema'
+require_dependency 'api/v3/work_packages/schema/specific_work_package_schema'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.